src/test/java/api/loans/agetolost/ScheduledAgeToLostFeeChargingApiTest.java

```diff
@@ -76,9 +77,12 @@ public void shouldChargeItemFee() {
   public void shouldChargeItemProcessingFee() {
     final double expectedProcessingFee = 99.54;
 
-    val policy = lostItemFeePoliciesFixture
-      .ageToLostAfterOneMinutePolicy()
+    val policy = new LostItemFeePolicyBuilder()
+      .withName("shouldChargeItemProcessingFee")
+      .withItemAgedToLostAfterOverdue(Period.weeks(1))
+      .withPatronBilledAfterAgedLost(Period.weeks(2))
       .withNoChargeAmountItem()
+      .doNotChargeProcessingFee()
```

Comment:
This change suggests that the processing fee should not be charged. My interpretation of the assertions (which have not changed) is that an item processing fee is still charged.

Please can you help me understand the change in behaviour in this change?

Reply:
In lost item policy we have a field to set processing fee amount and two flags to specify whether to charge processing fee when item is declared lost and whether to charge processing fee when item is aged to lost:

![image](https://user-images.githubusercontent.com/9094650/92399222-9c3b1d00-f132-11ea-8af3-9649b792ab1a.png)

`doNotChargeProcessingFee` - means do not charge processing fee when item is declared lost.
`chargeItemAgedToLostProcessingFee` - means charge processing fee when item is aged to lost.

Reply:
Thank you for that explanation.

It's unfortunate that the properties in the policies are named that way.

Could the intent of them be better reflected by changing the name of the builder methods e.g. `doNotChargeProcessingFeeWhenDeclaredLost` instead of `doNotChargeProcessingFee`?

Is there a test that demonstrates the impact of these changes, for example, is there a test that demonstrates for both of these settings that the item processing fee is not charged even if an amount is specified?

If not, please could tests be added (unit if possible).

Reply:
Done. I have added unit tests and refactored methods to reflect different type of processing fees.